src/datasets/load.py

```diff
@@ -1772,31 +1773,30 @@ def dataset_module_factory(
                 else:
                     raise e
             if filename in [sibling.rfilename for sibling in dataset_info.siblings]:  # contains a dataset script
-                if config.USE_PARQUET_EXPORT:
+                fs = HfFileSystem(endpoint=config.HF_ENDPOINT, token=download_config.token)
+                with fs.open(f"datasets/{path}/{filename}", "r", revision=revision, encoding="utf-8") as f:
+                    can_load_config_from_parquet_export = (
+                        "DEFAULT_CONFIG_NAME" not in f.read() or _ignore_default_config
+                    )
```

Comment:
I think it would be better to refactor this code and avoid the network call and file read if the variable "_ignore_default_config" is True.

Reply:
done !